ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -251,10 +251,14 @@ cat > "${rsyncd_ignition_patch}" << EOF
 EOF
 
 # patch rsync setting to ignition
-patch_ignition_file "${bastion_ignition_file}" "${rsyncd_ignition_patch}"
+if [[ "${CLUSTER_TYPE}" == "ibmcloud" ]]; then
```

Comment:
If I remember right, the rsync patch is required by https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L271, once skip the patch, the rsync maybe not working, that would lead `gather-installer-boostrap-logs-in-bastion` failed accordingly. That means once ibmcloud private cluster install failed, no installer gather log would be collecdted.

Reply:
What is the root cause ?